charts/payone-integration/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+name: payone-integration
```

Comment:
we can call it payment-integration as it supposed to be used for both

Reply:
[808a35ee](https://github.com/commercetools/k8s-charts/commit/808a35ee01fa2079fd24ffcd558eae3889a074ec#diff-e48b4288238229ccd81330e08764575dR2)